README.md

```diff
@@ -1,10 +1,10 @@
 # Rack::Test
 [![Gem Version](https://badge.fury.io/rb/rack-test.svg)](https://badge.fury.io/rb/rack-test)
-[<img src="https://travis-ci.org/rack-test/rack-test.svg?branch=master" />](https://travis-ci.org/rack-test/rack-test)
+[<img src="https://travis-ci.org/rack/rack-test.svg?branch=master" />](https://travis-ci.org/rack-test/rack-test)
```

Comment:
Travis support no longer works and can be removed.  We should either remove or fix the Code Climate and Test Coverage buttons to point to the correct repository.

Reply:
I'm generally in favour of removing all badges except GH Actions status (even that I don't feel strongly about).